akka-http-core/src/main/scala/akka/http/impl/engine/client/pool/NewHostConnectionPool.scala

```diff
@@ -191,6 +190,7 @@ private[client] object NewHostConnectionPool {
                           updateState(_.onTimeout(this))
                         }
                       })
+                  case _ ⇒
```

Comment:
add a comment that it's intentionally ignoring? Can be confusing otherwise in rebases when conflicts happen etc..

Reply:
Thanks, good point, added comment.